### ISSUE TITLE
SID task loss function. 

### ISSUE THREAD

**Issue by raotnameh** [s3prl/s3prl#453]
ASV and SID tasks are very similar and yet have different loss functions.  ASV has AMsoftmax, and SID has softmax loss function, respectively. 

Why was this choice made? Furthermore, changing the loss function is acceptable or not?

**Comment by raotnameh** [comment]
@leo19941227
